### PR TITLE
Update PanelGroup layout to match Credit Application v2 design

### PR DESCRIPTION
**Pull request by rruiz85**
Updates `PanelGroup` scss layout

### CODE REVIEW

File: components/PanelGroup/PanelGroup.jsx [hunk 1]
```diff
@@ -90,7 +90,7 @@ export default class PanelGroup extends React.Component<PanelGroupProps> {
 
   render() {
     const { children, id, inner, className } = this.props
-    const toggleIconName = inner ? "angle-up" : "caret-up"
+    const toggleIconName = inner ? "angle-up" : "chevron-down "
```
Comment (drborges) [inline components/PanelGroup/PanelGroup.jsx]: Shouldn't we use `chevron-up` and `chevron-down` pairs?

File: components/DOBInput/styles.scss.flow [hunk 2]
```diff
@@ -0,0 +1,7 @@
+// @flow
+/* This file is automatically generated by css-modules-flow-types */
+declare module.exports: {|
+  +'dob': string;
+  +'hidden-xxs': string;
+  +'visible-xxs': string;
+|};
```
Comment (thatjsguy) [inline components/DOBInput/styles.scss.flow]: You can delete this is you want. These are useless.

**Comment by thatjsguy** [comment]
@rruiz85 How much of your own whole app regression testing have you done locally against these changes?

**Comment by jasoncypret** [comment]
@thatjsguy Should we close this PR? Looks like these changes were added elsewhere?

**Pull request merged**